app-games/src/main/java/com/aptoide/android/aptoidegames/AptoideApplication.kt

```diff
@@ -5,8 +5,8 @@ import android.content.Context
 import androidx.datastore.core.DataStore
 import androidx.datastore.preferences.core.Preferences
 import androidx.datastore.preferences.preferencesDataStore
-import com.aptoide.android.aptoidegames.installer.notifications.InstallerNotificationsManager
 import cm.aptoide.pt.install_manager.InstallManager
+import com.aptoide.android.aptoidegames.installer.notifications.InstallerNotificationsManager
```

Comment:
This single change does not need a separate commit, since it's just a single line. This commit can be dropped and this change goes into another commit